src/main/java/org/reactome/release/qa/check/HumanReactionsWithoutDiseaseAndHaveNonHumanPhysicalEntitiesCheck.java

```diff
@@ -16,18 +17,17 @@
 @SliceQACheck
 public class HumanReactionsWithoutDiseaseAndHaveNonHumanPhysicalEntitiesCheck extends AbstractQACheck {
 
-    private static final String INNATE_IMMUNITY_PATHWAY_DBID = "168249";
-    private List<String> skiplistDbIds = new ArrayList<>();
+    private SkipList skipList;
 
     @Override
     public QAReport executeQACheck() throws Exception {
         QAReport report = new QAReport();
-        this.skiplistDbIds.add(INNATE_IMMUNITY_PATHWAY_DBID);
+        skipList = new SkipList(this.getDisplayName());
 
         Collection<GKInstance> reactions = dba.fetchInstancesByClass(ReactomeJavaConstants.ReactionlikeEvent);
         for (GKInstance reaction : reactions) {
             // isHumanDatabaseObject checks that the species attribute only contains a Homo sapiens species instance. Multi-species RlEs are excluded.
-            if (!QACheckerHelper.memberSkipListPathway(reaction, skiplistDbIds)
+            if (!QACheckerHelper.memberSkipListPathway(reaction, skipList.getSkipList())
```

Comment:
Maybe this method could be called “getSkipListDbIds”

Reply:
Done